### PR TITLE
Add Event's cancelBubble attribute

### LINKED ISSUE #211: cancelBubble legacy property
[`UIEvent.cancelBubble`](https://developer.mozilla.org/en-US/docs/Web/API/UIEvent/cancelBubble) works in all modern browsers.
Testcase: http://jsbin.com/jufexu/edit?html,css,js,output

[`Event.cancelBubble`](https://developer.mozilla.org/en-US/docs/Web/API/Event/cancelBubble) works in all modern browsers (Chrome, Safari, Edge, IE11) except Firefox.
Testcase: http://jsbin.com/wirelo/edit?html,js,output

To accurately reflect reality, I propose speccing at least one of these properties.

### PR DESCRIPTION
**Pull request by annevk**
More delightful garbage from the past. Fixes #211.

### CODE REVIEW

**Comment by annevk** [comment]
@cvrebert do you have tests for this already?

**Comment by cvrebert** [comment]
@annevk Yes. Though actual submission of them is blocked on the web-platform-tests equivalent of https://github.com/w3c/csswg-test/pull/1091

File: dom.bs [hunk 1]
```diff
@@ -511,6 +511,7 @@ interface Event {
   readonly attribute unsigned short eventPhase;
 
   void stopPropagation();
+           attribute boolean cancelBubble; // historical
```
Comment (cvrebert) [inline dom.bs]: Indentation seems wrong?

Comment (annevk) [inline dom.bs]: This is the normal indentation for attributes without readonly. They are aligned with their readonly counterparts.

**Comment by annevk** [comment]
@cvrebert great, is there an issue on file for WPT to get such a LICENSE resource?

**Comment by cvrebert** [comment]
@annevk Just filed https://github.com/w3c/web-platform-tests/issues/4295

**Comment by cvrebert** [comment]
> The {{Event/cancelBubble}} attribute's setter must set <a>context object</a>'s <a>stop propagation flag</a> if the given value is true, and unset it otherwise.

Erm, if we want to make it equivalent to `stopPropagation` (per the issue), then it shouldn't be possible to unset it, except via (re)initialization. So the setter shouldn't do anything if the value is not `true`.

**Comment by cvrebert** [comment]
Got the OK from Legal, so expect a testcases PR tonight.

**Comment by cvrebert** [comment]
Proposed tests: https://github.com/w3c/web-platform-tests/pull/4304

**Comment by annevk** [comment]
Cool, guess we should wait and see if any browsers are comfortable restricting the semantics further as you suggested.

**Pull request merged**